src/libcore/num/dec2flt/mod.rs

```diff
@@ -232,8 +234,7 @@ fn convert<T: RawFloat>(mut decimal: Decimal) -> Result<T, ParseFloatError> {
         return Ok(x);
     }
     // AlgorithmM and AlgorithmR both compute approximately `f * 10^e`.
-    let max_digits = decimal.integral.len() + decimal.fractional.len() +
-                     decimal.exp.abs() as usize;
+    let max_digits = decimal.integral.len() + decimal.fractional.len() + decimal.exp.abs() as usize;
```

Comment:
Oh, right, it's exactly 100 chars. I pasted the line (without leading `+`) into a file and let the editor tell me the cursor position after the semicolon, but that is off by one. However, 100 chars still conflicts with the official rule that [lines must not exceed 99 characters][99chars]. I don't know if tidy will follow that rule or accept 100 lines, but in any case it bugs me.

[99chars]: http://doc.rust-lang.org/nightly/style/style/whitespace.html